lib/viewing/HasChildViews.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-var _ = require("lodash");
+var _ = require("underscore");
```

Comment:
You can remove underscore entirely in this file by just using `typeof thing === "function"`